third_party/conan/lockfiles/linux/clang7_debug/conan.lock

```diff
@@ -306,44 +425,868 @@
      "35",
      "32",
      "8"
+    ],
+    "build_requires": [
+     "103"
     ]
    },
    "35": {
-    "pref": "glew/2.1.0@orbitdeps/stable#0:20fac438b2131dfc2a10463e5e35a8f1c508e016#0",
-    "options": "fPIC=True\nshared=False\nsystem_mesa=True"
+    "pref": "glew/2.1.0@orbitdeps/stable#0:20fac438b2131dfc2a10463e5e35a8f1c508e016",
+    "options": "fPIC=True\nshared=False\nsystem_mesa=True",
+    "build_requires": [
+     "55"
+    ]
    },
    "36": {
-    "pref": "libssh2/1.9.0#df2b6034da12cc5cb68bd3c5c22601bf:3edd7a996ca4d783d357da87bcb97f8517572791#950373b497ed790a458785afcfdc398f",
+    "pref": "libssh2/1.9.0#df2b6034da12cc5cb68bd3c5c22601bf:3edd7a996ca4d783d357da87bcb97f8517572791",
     "options": "crypto_backend=openssl\nenable_crypt_none=False\nenable_mac_none=False\nfPIC=True\nshared=False\nwith_zlib=True\nopenssl:386=False\nopenssl:fPIC=True\nopenssl:no_asm=False\nopenssl:no_async=False\nopenssl:no_bf=False\nopenssl:no_cast=False\nopenssl:no_des=False\nopenssl:no_dh=False\nopenssl:no_dsa=False\nopenssl:no_dso=False\nopenssl:no_hmac=False\nopenssl:no_md2=False\nopenssl:no_md5=False\nopenssl:no_mdc2=False\nopenssl:no_rc2=False\nopenssl:no_rc4=False\nopenssl:no_rc5=False\nopenssl:no_rsa=False\nopenssl:no_sha=False\nopenssl:no_sse2=False\nopenssl:no_threads=False\nopenssl:no_zlib=False\nopenssl:openssldir=None\nopenssl:shared=False\nzlib:fPIC=True\nzlib:minizip=False\nzlib:shared=False",
     "requires": [
      "8",
      "9"
+    ],
+    "build_requires": [
+     "97"
     ]
    },
    "37": {
-    "pref": "imgui/1.69@bincrafters/stable#0:8af60a965d2b176751abd5c14af8f4f7530f2c71#0",
-    "options": "fPIC=True\nshared=False"
+    "pref": "imgui/1.69@bincrafters/stable#0:8af60a965d2b176751abd5c14af8f4f7530f2c71",
+    "options": "fPIC=True\nshared=False",
+    "build_requires": [
+     "57"
+    ]
    },
    "38": {
-    "pref": "grpc_codegen/1.27.3@orbitdeps/stable#ec39b3cf6031361be942257523c1839a:0b6eee9c211529a5558295428b69441f41a01ae6#b1742c11862e056eec8187956508027c",
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
+   "39": {
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
+   "40": {
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
+   "41": {
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
+   "42": {
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
+   "43": {
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
+   "44": {
+    "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
+    "options": ""
+   },
```

Comment:
The nodes from 39 to 44 are the same. Is it expected?

Reply:
Yes. Every package get it's own node with build requirements.